SignalServiceKit/src/Messages/Attachments/TSAttachmentStream.m

```diff
@@ -345,10 +351,17 @@ - (CGSize)calculateImageSize
             = (__bridge_transfer NSDictionary *)CGImageSourceCopyPropertiesAtIndex(source, 0, (CFDictionaryRef)options);
         CGSize imageSize = CGSizeZero;
         if (properties) {
+            NSNumber *orientation = properties[(NSString *)kCGImagePropertyOrientation];
```

Comment:
A critical fix.  Our image sizing logic didn't take into account orientation.

Reply:
Nope, see the "version migration" done above.

Reply:
Do we want to persist the migrated data so that it isn't being migrated at run time every time an old attachment is initWithCoder?

Reply:
Yup, but I think the existing logic will take care of that.  See `[latestInstance saveWithTransaction:transaction];`